app/services/support_interface/application_monitor.rb

```diff
@@ -0,0 +1,21 @@
+module SupportInterface
+  class ApplicationMonitor
```

Comment:
Once an application has finished its life — ie enrolled, declined, withdrawn, etc, should we filter it out of these queries?

Reply:
Yeah, that's a good point. Pushed a commit.